pvlib/bifacial/loss_models.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import pandas as pd
+
+
+def power_mismatch_deline(
+    rmad,
+    model=(0, 0.142, 0.032),
+    fill_factor: float = None,
+    fill_factor_reference: float = 0.79,
+):
+    r"""
+    Estimate DC power loss due to irradiance non-uniformity.
+
+    This model is described for bifacial modules in [1]_, where the backside
+    irradiance is less uniform due to mounting and site conditions.
+
+    The power loss is estimated by a polynomial model of the Relative Mean
+    Absolute Difference (RMAD) of the cell-by-cell total irradiance.
+
+    Use ``fill_factor`` to account for different fill factors between the
+    trained model and the module of interest. Specify model's fill factor with
+    ``fill_factor_reference``.
+
+    .. versionadded:: 0.11.1
+
+    Parameters
+    ----------
+    rmad : numeric
+        The Relative Mean Absolute Difference of the cell-by-cell total
+        irradiance. [Unitless]
+        Check out the *Notes* section for the equation to calculate it from the
+        bifaciality and the front and back irradiances.
+
+    model : float collection or numpy.polynomial.polynomial.Polynomial, default ``(0, 0.142, 0.032)``
+        The model coefficients to use.
+
+        If a :external:class:`numpy.polynomial.polynomial.Polynomial`,
+        it is evaluated as is. If not a ``Polynomial``, it must be the
+        coefficients of a polynomial in ``rmad``, where the first element is
+        the constant term and the last element is the highest order term. A
+        :external:class:`~numpy.polynomial.polynomial.Polynomial`
+        will be created internally.
+
+    fill_factor : float, optional
+        Fill factor at standard test condition (STC) of the module.
+        Accounts for different fill factors between the trained model and the
+        module under non-uniform irradiance.
+        If not provided, the model is used as is.
+
+    fill_factor_reference : float, default 0.79
+        Fill factor at STC of the module used to train the model.
+        If ``fill_factor`` is provided, this value is used to scale the output
+        of the model.
+
+    Returns
+    -------
+    loss : numeric
+        The fractional power loss. [Unitless]
+        Output will be a ``pandas.Series`` if ``rmad`` is a ``pandas.Series``.
+
+    Notes
+    -----
+    The default model implemented is equation (11) [1]_:
+
+    .. math::
+
+       M[\%] = 0.142 \Delta[\%] + 0.032 \Delta[\%]^2 \qquad & \text{(11)}
+
+    where :math:`\Delta[\%]` is the Relative Mean Absolute Difference of the
+    global irradiance, Eq. (4) of [1]_ and [2]_.
+
+    The losses definition is Eq. (1) of [1]_, and it's defined as a loss of the
+    output power:
+
+    .. math::
+
+       M[\%] = 1 - \frac{P_{Array}}{\sum P_{Cells}} \qquad \text{(1)}
```

Comment:
This would need a `\times 100\%` to be correct, I think.  But since this function has unitless inputs and outputs, maybe we should get rid of the `[%]` everywhere anyway?